reference/constraints/NoSuspiciousCharacters.rst

```diff
@@ -0,0 +1,149 @@
+NoSuspiciousCharacters
+======================
+
+A good thing with Unicode is that it allows to use (almost) every imaginable character.
+A bad thing is that this makes spoofing rather easy: "symfony.com" and "ѕymfony.com"
+look similar, but the latter actually starts with a `cyrillic small letter dze`_.
```

Comment:
```suggestion
Unicode allows to use (almost) every imaginable character, including characters that look alike:
"symfony.com" and "ѕymfony.com" look similar, but the latter actually starts with a `cyrillic small letter dze`_,
an attacker could redirect users to a different URL than the user expects, which would permit spoofing attacks.
```

Reply:
Not a fan of the repetition (look alike/look similar) :confused: 

Note that I drafted this PR since some of the constraint’s constants disappeared. I’ll mark it ready for review when I finish updating it.

Reply:
Right, I edited the comment above.

Reply:
imaginable character/indiscernible characters :sweat_smile: 

Also it seems the user being "redirected" to the attacker’s URL means the spoofing attack already succeeded?

Reply:
I tried to add a real-world example but I can't find a simple sentence.

Reply:
I updated the introduction; does it look better to you?

Reply:
I really like this new introduction. :+1: Thanks for the update.